tests/kafkatest/tests/client/quota_test.py

```diff
@@ -204,7 +204,7 @@ def validate(self, broker, producer, consumer):
                    (producer_maximum_bps, producer_quota_bps, self.maximum_client_deviation_percentage)
 
         # validate that maximum_broker_byte_in_rate <= producer_quota * (1 + maximum_broker_deviation_percentage/100)
-        broker_byte_in_attribute_name = 'kafka.server:type=BrokerTopicMetrics,name=BytesInPerSec:OneMinuteRate'
+        broker_byte_in_attribute_name = 'org.apache.kafka.storage.log.metrics:type=BrokerTopicMetrics,name=BytesInPerSec:OneMinuteRate'
```

Comment:
That will be a issue about compatibility. Maybe we should make `KafkaMetricsGroup` take custom `pkg` and `simpleName` to keep the metrics compatibility